File: docs/code_examples/playwright/javascript/firefox_connect.js
import { firefox } from 'playwright';

const browser = await firefox.connect({
    wsEndpoint: 'wss://playwright.xclouds.dev/v1.58/?api_key=YOUR_API_KEY',
});
const page = await browser.newPage();
await page.goto('https://example.com');
await browser.close();

File: docs/code_examples/playwright/csharp/firefox_connect.cs
using Microsoft.Playwright;

using var playwright = await Playwright.CreateAsync();
var browser = await playwright.Firefox.ConnectAsync(
    "wss://playwright.xclouds.dev/v1.58/?api_key=YOUR_API_KEY"
);
var page = await browser.NewPageAsync();
await page.GotoAsync("https://example.com");
await browser.CloseAsync();
